iothub/service/src/Microsoft.Azure.Devices.csproj

```diff
@@ -164,7 +164,7 @@
 
   <ItemGroup>
     <!-- FXCop -->
-    <PackageReference Condition=" '$(Configuration)' == 'Debug' " Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.8">
+    <PackageReference Condition=" '$(Configuration)' == 'Debug' " Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.1">
```

Comment:
Track2 libraries are only on 2.6.2. Do you mind starting a convo with Pavel Krymets about why they haven't updated to the latest?

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Packages.Data.props#L66

Reply:
These have been updated